shotover-proxy/tests/cassandra_int_tests/prepared_statements.rs

```diff
@@ -73,26 +73,26 @@ async fn select_cross_connection<Fut>(
 async fn use_statement(session: &CassandraConnection) {
     // Create prepared command with the correct keyspace
     run_query(session, "USE test_prepare_statements;").await;
-    let _prepared = session.prepare("INSERT INTO table_1 (id) VALUES (?);");
+    let prepared = session.prepare("INSERT INTO table_1 (id) VALUES (?);");
 
     // change the keyspace to be incorrect
     run_query(session, "USE test_prepare_statements_empty;").await;
 
     // TODO: Somehow executing the query fails when run through shotover but succeeds when run directly against cassandra
```

Comment:
huh, I have no idea what we changed to fix this test.
Considering it is fixed though, we should remove this TODO.